examples/v1beta1/trial-images/enas-cnn-cifar10/RunTrial.py

```diff
@@ -76,18 +76,21 @@
     y_train = to_categorical(y_train)
     y_test = to_categorical(y_test)
 
-    augmentation = ImageDataGenerator(
-        width_shift_range=0.1,
-        height_shift_range=0.1,
-        horizontal_flip=True)
+    augmentation = tf.keras.Sequential([
+        Rescaling(1./255),
+        RandomFlip('horizontal'),
+        RandomTranslation(height_factor=0.1, width_factor=0.1),
+    ])
```

Comment:
This is a deprecated approach and moved to `tensorflow.keras.preprocessing.image.

```shell
Traceback (most recent call last):
  File "/opt/enas-cnn-cifar10/RunTrial.py", line 19, in <module>
    from keras.preprocessing.image import ImageDataGenerator
ImportError: cannot import name 'ImageDataGenerator' from 'keras.preprocessing.image' (/usr/local/lib/python3.11/site-packages/keras/preprocessing/image/__init__.py)
```

https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/image/ImageDataGenerator

So, I introduced the recommended approach.